s4e-backend/src/main/java/pl/cyfronet/s4e/controller/ConfigController.java

```diff
@@ -67,10 +67,10 @@ public ConfigResponse config() {
     @ApiResponses({
             @ApiResponse(responseCode = "200", description = "OK")
```

Comment:
I see that you're using user data inside of controller, but you don't included the 401, 403 errors in the Api responses. Does it have a purpose?

Reply:
Here omitting 401/403 is intentional, as these endpoints should always return 200.
Product filtering in the supplier only narrows down the list of returned products.